environment.yml

```diff
@@ -0,0 +1,14 @@
+name: galaxy_training_material
+channels:
+- bioconda
+- r
```

Comment:
r is not needed.
conda-forge should be before defaults.
actually I'm not sure bioconda is needed :)